src/transformers/models/flaubert/modeling_flaubert.py

```diff
@@ -167,6 +167,40 @@ def forward(
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
     ) -> Union[Tuple, BaseModelOutput]:
+        r"""
```

Comment:
Thanks a lot for the PR @abdouaziz ! 

I think for now the most important doc example to fix here are `ForSequenceClassification` , `ForTokenClassification`, `ForQuestionAnswering` and `ForMaskedLM`. Could you maybe just add the correct values to the `@add_code_sample_docstring(...)` as is done here: https://github.com/huggingface/transformers/pull/16363/files#r833804669 - no need to overwrite the existing doc string normally :-)

Also please don't forget to add the modeling file to the documentation tests afterwards: https://github.com/huggingface/transformers/pull/16363/files#r833804872

Reply:
This guide might also help on how to find good checkpoitns: https://github.com/huggingface/transformers/issues/16292#issuecomment-1076919075 :-)

Reply:
Thanks @patrickvonplaten  for the clarification. I have one last question for the part of Flaubert. according to your example the forward functions are defined. For the classes of `ForSequenceClassification` , `ForTokenClassification`, `ForQuestionAnswering` and `ForMaskedLM` have just the real value on the @add_code_sample_docstring(...) which makes an example like following for ROBERTA: 

@add_start_docstrings_to_model_forward(ROBERTA_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
         checkpoint="cardiffnlp/twitter-roberta-base-emotion",
         output_type=SequenceClassifierOutput,
         config_class=_CONFIG_FOR_DOC,
         expected_output="'optimism'",
         expected_loss=0.08,
     )

But in the case of FLAUBERT as the forward function is not defined for the classes `ForSequenceClassification` , `ForTokenClassification`, `ForQuestionAnswering` and `ForMaskedLM`.

My question : we add only @add_code_sample_docstrings(....) ? 
Or we add @add_start_docstrings_to_model_forward(....) and 
@add_code_sample_docstrings(....)

@ydshieh 

> Thanks a lot for the PR @abdouaziz !
> 
> I think for now the most important doc example to fix here are `ForSequenceClassification` , `ForTokenClassification`, `ForQuestionAnswering` and `ForMaskedLM`. Could you maybe just add the correct values to the `@add_code_sample_docstring(...)` as is done here: https://github.com/huggingface/transformers/pull/16363/files#r833804669 - no need to overwrite the existing doc string normally :-)
> 
> Also please don't forget to add the modeling file to the documentation tests afterwards: https://github.com/huggingface/transformers/pull/16363/files#r833804872

Reply:
@abdouaziz, after discussing with the team: 

since `FlauBERT`  architecture is exactly the same as `XLM`, the code in `FlauBERT` is meant to be just an alias, and should be left as it is.

We should prepare the model list for this sprint in a better way, sorry for this inconvenience.
Would you like to look into another architecture? I will try to update the list in the next few hours.

Thank you again!

Reply:
Thanks for the feedback I will choose others then . 
Thanks

Reply:
Very sorry about the confusion here @abdouaziz, I missed the fact that FlauBERT is just an alias of another model. Very sorry about being not being very clear here